compiler/rustc_codegen_llvm/src/coverageinfo/map_data.rs

```diff
@@ -1,5 +1,4 @@
 use crate::coverageinfo::ffi::{Counter, CounterExpression, ExprKind};
-
```

Comment:
These whitespace changes seem to be left over from an earlier draft. Please remove them.